templates/features/high-availability.html

```diff
@@ -0,0 +1,87 @@
+        </p>
+        <p>
+         MicroK8s makes high availability resilient and self-healing with no administrative interventions required.
+        </p>
+      </div>
+      <div class="col-4 u-hide--small">
+        <img src="https://assets.ubuntu.com/v1/9d785fad-Automatic+node+allocation.svg" width="230" alt="Automatic node allocation"/>
+      </div>
+    </div>
+    <div class="row">
+      <h2>What is high availability Kubernetes?</h2>
+      <div class="col-8">
+        <p>
+          A highly available Kubernetes cluster can withstand a failure on any component and continue serving workloads without interruption. Three factors are necessary for a highly available Kubernetes cluster:
+        </p>
+        <ul class="p-list">
+          <li class="p-list__item is-ticked">There <b>must be more than one worker node</b>. Since MicroK8s uses every node as a worker node, there is always more than one worker if there is more than one node in the cluster.</li>
+          <li class="p-list__item is-ticked">The <b>Kubernetes API services must be running on more than one node</b> so that losing a single node would not render the cluster inoperable. Every node in the MicroK8s cluster is an API server, which simplifies load-balancing and means we can switch instantaneously to a different API endpoint if one fails.</li>
+          <li class="p-list__item is-ticked"><b>The cluster state must be in a reliable datastore</b>. By default, MicroK8s uses <a class="p-link--external" href="https://dqlite.io/">Dqlite</a>, a high-availability SQLite, as its datastore.</li>
+        </ul>
+        <p>
+          All that is required for HA MicroK8s is three or more nodes in the cluster, at which point Dqlite is automatically highly available. If the cluster has more than three nodes, then additional nodes will be standby candidates for the datastore and promoted automatically if the datastore loses one of its nodes. The automatic promotion of standby nodes into the voting cluster of Dqlite makes MicroK8s HA autonomous and ensures that quorum is maintained even if no administrative action is taken.
+        </p>
+      </div>
+    </div>
+    <div class="row">
+      <h2>HA Kubernetes cluster with MicroK8s</h2>
+      <div class="col-8">
+        <p>
+          MicroK8s delivers a production-grade Kubernetes cluster simply by adding more MicroK8s nodes. There is no extra configuration required - install MicroK8s on three machines, run the join command to link them together and in moments you have a production-grade Kubernetes cluster with HA enabled automatically.
+        </p>
+        <p class="u-sv3">
```

Comment:
Could you file a vanilla bug for this please. The fact videos do not have enough padding.

Reply:
Will do